packages/components/src/components/loading/loading-dtrader-v2.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PositionsLoader from './dtrader-v2-loaders/positions-loader';
+import ContractDetailsLoader from './dtrader-v2-loaders/contract-details-loader';
+import TradeLoader from './dtrader-v2-loaders/trade-loader';
+import HeaderLoader from './dtrader-v2-loaders/header-loader';
+
+type TLoadingDTraderV2Props = {
+    initial_app_loading?: boolean;
+    is_closed_tab?: boolean;
+    is_contract_details?: boolean;
+    is_header?: boolean;
+    is_positions?: boolean;
+};
+
+const LoadingDTraderV2 = ({
```

Comment:
I don't this component will be used anywhere else. Can we move this component to dtrader AppV2?

Reply:
Skeleton can stay in the components package but we should move the dtrader specific loaders to the dtrader package

Reply:
we can move HeaderLoader to core

Reply:
@balakrishna-deriv  we cannot move the dtrader-specific loaders to dtrader AppV2 (trader package) because we need to load them from core/ in `packages/core/src/App/Components/Routes/binary-routes.jsx`

Reply:
i'll leave a comment that we need these loaders in /components temprorarily, and I'll put HeaderLoader in core :)

Reply:
done👌🏻